awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx

```diff
@@ -101,7 +101,9 @@ function InventoryListItem({
         <DataListItemCells
           dataListCells={[
             <DataListCell key="sync-status" isIcon>
-              <SyncStatusIndicator status={syncStatus} />
+              {inventory.kind !== 'smart' ? (
```

Comment:
I think using the logical && operator would be better here. More concise and a bit easier to ready. so this line would be `inventory.kind ==='smart' &&  <SyncStatusIndicator status={syncStatus} />`

Reply:
I can change it, but the logic that you suggested is the opposite of what we need.